packages/contracts/test/router/v6/sudoswap/offers.test.ts

```diff
@@ -0,0 +1,93 @@
+import { Contract } from "@ethersproject/contracts";
+import { parseEther } from "@ethersproject/units";
+import * as Sdk from "../../../../../sdk/src";
```

Comment:
Can use `import * as Sdk from "@reservoir0x/sdk/src"`

Reply:
i think i've gotta use the local version of the sdk, rather than `@reservoir0x` here, 'cause it relies on code i've implemented in this branch, like this
```
let swapList = new Sdk.Sudoswap.SwapList(swapListPair, swapListNftIds);

let sudoswap = new Sdk.Sudoswap.Router(chainId);
let data = sudoswap.swapETHForSpecificNFTsTxData([swapList], ethRecipient, nftRecipient);
```